### DISCUSSION TITLE
Region based quest to check for green waste containers?

### DISCUSSION THREAD

**Discussion by mahdi1234** [streetcomplete/StreetComplete#3091] [General]
Hi guys,

Before creating issue I'd like to ask whether this fits SC or not.

In my town there's been established around 1200 green waste containers (added to existing recycling points) during spring out of which only fraction is mapped today.

Would it make sense for SC to re-survey all amenity=recycling without green waste tag in that region (which don't have check_date/survey:date after spring)?

cheers

**Comment by mnalis** [comment]
I guess this boils down to few questions:

- does StreetComplete implement resurvey for recycling container types, and if not, could/should it be added?
  (I'm not sure if it does already, but adding resurvey for this might be useful generally if it does not already exists)

- is community in your country (eg. mailing list for your country, or other "official"/preferred  communication channel) OK with mass-editing of `check_date` tag in that town (so one could set it enough in the past for elements that do not have it already, so above-mentioned resurvey would happen automatically)? Of course, it should be discussed what are inclusion/exclusion rules (in order not to force quest on nodes created after start of the spring, for example), and who would do it.

-  are there enough people in the area willing to do the resurvey on all those containers?

**Reply by mahdi1234** [reply]
I can answer 2 and 3

- if it should imply inserting such a tag I would definitely ask (active member of country mail list), but I thought of that SC has right now country-specific tasks, so it could be limited to area without extra key (but again it's more question whether this is feasible for SC)

- given region is very active with mappers in general and SC as well (+ I would advertise on mail list/matrix channel)

**Reply by mnalis** [reply]
@mahdi1234 if the community is **not ok** with 2 and 3, than the point is moot anyway...

But if the community is OK with that, than you can check answer for 1. question yourself by simply trying (modify `check_date` to few years in the past for one container you know has changed), force-reload in StreetComplete and see if it is offered for solving/updating again:

- if it is offered again and you can answer that it has new recycling biowaste container, then it's all solved and you can proceed with you idea (with your community blessing)
- if it is not offered again, it indicates that the resurvey quest for that tag does not exists; so to me it looks like a valid new request / issue to ask for "please ask for resurvey on  recycling quest" (StreetComplete has several similar resurvey quests already, where it made sense).

**Reply by mahdi1234** [reply]
So it seems to behave like this:

- if the item has check_date older than two years it will re-trigger the check whether the recycling still exists, but not re-survey what kind of material can be recycled
- if the item as such was last changed more than two years ago it does the same

likely this is not yet implemented. I will log issue for it later.
